Assignment-2/src/main/java/application/MainClass.java

```diff
@@ -0,0 +1,73 @@
+package application;
+
+import java.util.List;
+import java.util.Scanner;
+
+import models.Display;
+import models.UserClass;
+import services.ServicesHelper;
+import services.SortServiceHelper;
+
+/**
+ * Main class.
+ */
+public final class MainClass {
+  private MainClass() {
+
+  }
+
+  /**
+   * Main function.
+   * @param args represents args
+   */
+  public static void main(String[] args) {
+    final String filePath = "users.json";
+
+    final List<UserClass> users = ServicesHelper.loadUsers(filePath);
+
+    final List<Integer> rolls = ServicesHelper.loadRollNum(users);
+
+    final Display content = new Display();
```

Comment:
Don't need a separate class for printing static message. 
Create a class for all constant static messages and use that where ever you need to output any static message.

Reply:
ok will remove it